src/JUnit.Xml.TestLogger/JUnitXmlTestLogger.cs

```diff
@@ -208,6 +222,22 @@ public void Initialize(TestLoggerEvents events, Dictionary<string, string> param
                     Console.WriteLine($"JunitXML Logger: The provided Failure Body Format '{failureFormat}' is not a recognized option. Using default");
                 }
             }
+
+            if (parameters.TryGetValue(FileEncodingKey, out string fileEncoding))
+            {
+                if (string.Equals(fileEncoding.Trim(), "UTF8Bom", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should we consider `Enum.TryParse` to retrieve the provided option? We can possibly treat one of `utf8` or `utf8bom` as default, we can do only one if check.

Reply:
Yeah, I let this be pretty verbose. Maybe something for https://github.com/spekt/testlogger ?

Reply:
Sounds reasonable.